standup/api/admin.py

```diff
@@ -5,4 +5,8 @@
 
 @admin.register(SystemToken)
 class SystemTokenAdmin(admin.ModelAdmin):
-    pass
+    list_display = ('token', 'summary', 'enabled', 'contact', 'created')
+    list_editable = ('enabled',)
+    list_filter = ('enabled',)
+    readonly_fields = ('token',)
+    ordering = ('created',)
```

Comment:
Thank you!

For the record, I haven't actually tested any of the admin things I've thrown together figuring we'd fix it when someone looked at it with some knowledge of what they wanted to do there.